api/app/src/models/medical/organization.ts

```diff
@@ -1,14 +1,17 @@
 import { DataTypes, Sequelize } from "sequelize";
+import { Organization as OrganizationType } from "@metriport/api";
```

Comment:
I think our model shouldn't rely on external components.

I think we should define a domain type (pure TS class/type on the /domain folder) and create a zod Schema and DTO type for the API boundary ou the route level, converting from/to the domain there.

Then we can have the API SDK importing those files _from the route_, not the domain.

This allows us to change the domain w/o changing the API boundary/interface, and vice-versa, lets the API own its data/business logic, and minimizes redefining the API boundary/interface types by having the SDK importing those from the API route.

Thread on slack about this: https://metriport.slack.com/archives/C04DMKE9DME/p1677555122160669